plugins/mocha/readme.md

```diff
@@ -42,4 +44,23 @@ hooks:
   test:ci:
     - Eslint
     - Mocha
-```
\ No newline at end of file
+```
+
+### Reporting code coverage
+
+If you want to test the coverage of your code (i.e., how many lines of your code are executed by your unit tests) you can use the [Istanbul](https://istanbul.js.org) tool that integrates with Mocha. All you need to do is install the command line interface, [nyc](https://github.com/istanbuljs/nyc), by running
```

Comment:
**nit-pick (non-blocking)**: for me, `All you need to do` is not inclusive language (like saying `you just have to` or `simply do xyz` which assumes a level of ease). Instead of
```
All you need to do is install the command line interface, [nyc](https://github.com/istanbuljs/nyc), by running
```

I would prefer
```
To use Istanbul, install the command line interface, [nyc](https://github.com/istanbuljs/nyc), by running
```